README.md

```diff
@@ -188,6 +188,20 @@ Notice how some of the configuration values are wrapped in angle brackets (e.g.
             "verify_server": "cert"
         },
 
+        // Controls how the HTTP API communicates with the iRODS server.
+        //
+        // When set to true, the following applies:
+        // - Only APIs supported by iRODS 4.2.11 will be used.
```

Comment:
no mention of 4.2.12... is that intentional?

and if someone is running 4.2.10 or prior?

Reply:
Oh, missed that. It's supposed to specify the minimum version tested.

4.2.11 and later.

Reply:
Done.